Assets/Scripts/Networking/CustomNetworkManager.cs

```diff
@@ -46,8 +46,12 @@ public override void OnClientSceneChanged(NetworkConnection conn)
 
         void OnLoaderFinished()
         {
-            if (NetStatus.IsServer) // don't do anything on server
-                return;
+            if (NetStatus.IsServer)
```

Comment:
We already talked about this. CustomNetworkManager should not be aware of rcon.

Instead, create a new script, attach it to GameManager prefab (not scene object), and put `OnLoaderFinished` in this script.